hashicorp/boundary-ui
# chore: 🤖 update ember-cli-mirage

# Description
This PR updates `ember-cli-mirage` to v3.0.3. This is a minor change that will help with updating packages for this [dependabot alert](https://github.com/hashicorp/boundary-ui/security/dependabot/140). 





## Screenshots (if appropriate)

I saw this warning while running test. I removed the entry for fireworm from yarn.lock [here](https://github.com/hashicorp/boundary-ui/pull/2458/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR13025-R13027) and ran `yarn install` again. This cleared up the warning.
<img width="826" alt="Screenshot 2024-08-22 at 4 55 40 PM" src="https://github.com/user-attachments/assets/b68b5798-645f-4bd4-8d73-b33b23b49f56">



## How to Test

Please navigate around the Admin UI and desktop client Vercel builds. I have done local testing and have not see any issues and this [build](https://github.com/hashicorp/boundary-ui-releases/actions/runs/10529040740) was successful.

## Checklist
